File: Vectors/Assets/Scripts/Javascript/CrossProduct.js
#pragma strict

private var vector1 : Vector3;
private var vector2 : Vector3;
private var crossVector : Vector3;

function Start() {
	vector1 = new Vector3(5, 0, 5);
	vector2 = new Vector3(5, 0, -2.5);
	
	crossVector = Vector3.Cross(vector1, vector2);
	Debug.Log(crossVector);
	Debug.Log("Area of parallelogram: " + crossVector.magnitude);
}

function Update() {
	Debug.DrawLine(Vector3.zero, Vector3(5, 0, 5), Color.red);
	Debug.DrawLine(Vector3.zero, Vector3(-5, 0, -2.5), Color.blue);
	Debug.DrawLine(Vector3.zero, crossVector, Color.yellow);
}

File: Vectors/Assets/Scripts/C#/CrossProduct.cs
using UnityEngine;
using System.Collections;

public class CrossProduct : MonoBehaviour {
	private Vector3 vector1;
	private Vector3 vector2;
	private Vector3 crossVector;

	void Start() {
		vector1 = new Vector3(5f, 0f, 5f);
		vector2 = new Vector3(5f, 0f, -2.5f);
		
		crossVector = Vector3.Cross(vector1, vector2);
		Debug.Log(crossVector);
		Debug.Log("Area of parallelogram: " + crossVector.magnitude);
	}

	void Update() {
		Debug.DrawLine(Vector3.zero, new Vector3(5f, 0f, 5f), Color.red);
		Debug.DrawLine(Vector3.zero, new Vector3(-5f, 0f, -2.5f), Color.blue);
		Debug.DrawLine(Vector3.zero, crossVector, Color.yellow);
	}
}